build_library/check_root

```diff
@@ -177,9 +177,20 @@ def check_shebang():
                 line = fd.readline(80)
                 if not line.startswith(b"#!"):
                     continue
-                cmd = line[2:].rstrip().split(None,1)[0]
+                words = line[2:].rstrip().split(None, 1)
```

Comment:
A couple things, if either env or the prog have arguments this will fail. I think this will handle both cases:

```python
args = line[2:].rstrip().split(None, 3)
cmd = args.pop(0)
if cmd == '/usr/bin/env' and args:
    prog = args.pop(0)
    if prog == '-i' and args:
        progs = args.pop(0)
    cmd = '(env)/%s' % prog
```